<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="apasham-171222" ignored="0" name="Functional Test" tests="4" failures="1" timestamp="2019-09-13T16:35:09 PDT" time="0.129" errors="0">
  <testcase name="doUserRegister" time="0.018" classname="regression.TestCase1">
    <failure type="java.lang.AssertionError" message="User was not registered">
      <![CDATA[java.lang.AssertionError: User was not registered
at regression.TestCase1.doUserRegister(TestCase1.java:29)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- doUserRegister -->
  <testcase name="doLogin" time="0.0" classname="regression.TestCase1">
    <skipped/>
  </testcase> <!-- doLogin -->
  <testcase name="fourthTest" time="0.0" classname="regression.TestCase1">
    <skipped/>
  </testcase> <!-- fourthTest -->
  <testcase name="thirdTest" time="0.006" classname="regression.TestCase1"/>
</testsuite> <!-- Functional Test -->
